app/models/order.rb

```diff
@@ -1,5 +1,19 @@
 class Order < ApplicationRecord
+  include ActionView::Helpers::NumberHelper
```

Comment:
@drod1000 @Laszlo-JFLMTCO @Dpalazzari do we need this line here if we've included it in ApplicationRecord above?